Kaas/Controllers/QuotesController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Kaas\Controllers;
+
+use Illuminate\Http\Response;
+use Kaas\Models\Quote;
+use Laravel\Lumen\Routing\Controller as BaseController;
+
+/**
+ * Created by PhpStorm.
+ * User: joe
+ * Date: 24/09/2016
+ * Time: 16:40
+ */
+class QuotesController extends BaseController
+{
+
+    public static function getResponse() : Response {
```

Comment:
These shouldn't be static, the class will be instantiated by Lumen automatically if routes file is setup like this:

$app->get('user/{id}', 'UserController@show');

It won't make a big difference now, but in the future if we want to inject service providers it will make life easier.